app/models/manager_refresh/inventory/automation_manager.rb

```diff
@@ -0,0 +1,33 @@
+module ManagerRefresh::Inventory::AutomationManager
+  extend ActiveSupport::Concern
+  include Core
+
+  class Inventory < ManagerRefresh::Inventory::Core::Inventory
+    def credentials
```

Comment:
so if we need to overwrite this for a specific manager, how would that look like?